packages/core/src/internal/motion/utils.spec.ts

```diff
@@ -170,10 +166,9 @@ describe('Animation Helpers: ', () => {
       const didRun = await runPropertyAnimations(propMap, (component as unknown) as AnimatableElement);
       expect(didRun).toBe(true);
       removeTestElement(testElement);
-      done();
     });
 
-    it('does NOT run if there is an animation label associated with the property value but no animation in ClarityMotion', async done => {
+    it('does NOT run if there is an animation label associated with the property value but no animation in ClarityMotion', async () => {
```

Comment:
Jasmine deprecated the `done` function when used in an async function. This was the last of them and the warning will no longer be in the build console